### DISCUSSION TITLE
Docker swarm containers not showing in coroot

### DISCUSSION THREAD

**Discussion by Mehdi-BR** [coroot/coroot#251] [Q&A]
Hello there, 

We are currently facing an issue when integrating coroot with the docker swarm cluster . The problem is that coroot is not displaying most of the containers we have.
Upon inspecting the logs (linked down bellow) we noticed a line reoccurring multiple time which is :
W0519 14:44:03.981674  244111 registry.go:341] failed to create container pid=245494 cg=/system.slice/docker-67d75ce828cebdf472081eee

Our setup : 
we have a docker swarm cluster installed in multiple machines, in all of these machines we deployed coroot node agent as docker container pointing to a coroot instance deployed as container in a separate monitoring vm.

I0519 14:43:30.745462  244111 registry.go:314] calculated container id 244537 -> /system.slice/docker-e229b268f8788854725ae417b3a603e
c9d1f8b6dd393b4457cd26c1cbe82cbc6.scope -> /swarm/elk/filebeat/rq437f964bxm6zbf3d9zfst9v
W0519 14:43:30.745513  244111 registry.go:341] failed to create container pid=244537 cg=/system.slice/docker-e229b268f8788854725ae417
b3a603ec9d1f8b6dd393b4457cd26c1cbe82cbc6.scope id=/swarm/elk/filebeat/rq437f964bxm6zbf3d9zfst9v: no such file or directory
W0519 14:43:30.745513  244111 registry.go:341] failed to create container pid=244537 cg=/system.slice/docker-e229b268f8788854725ae417
b3a603ec9d1f8b6dd393b4457cd26c1cbe82cbc6.scope id=/swarm/elk/filebeat/rq437f964bxm6zbf3d9zfst9v: no such file or directory
I0519 14:43:30.746509  244111 registry.go:314] calculated container id 244539 -> /system.slice/docker-e229b268f8788854725ae417b3a603e
c9d1f8b6dd393b4457cd26c1cbe82cbc6.scope -> /swarm/elk/filebeat/rq437f964bxm6zbf3d9zfst9v
I0519 14:43:30.746797  244111 container.go:911] "started container logparser" cg="/system.slice/docker-e229b268f8788854725ae417b3a603
ec9d1f8b6dd393b4457cd26c1cbe82cbc6.scope"
I0519 14:43:30.746835  244111 registry.go:345] "detected a new container" pid=244539 cg="/system.slice/docker-e229b268f8788854725ae41
7b3a603ec9d1f8b6dd393b4457cd26c1cbe82cbc6.scope" id="/swarm/elk/filebeat/rq437f964bxm6zbf3d9zfst9v"
 
 
W0519 14:43:32.621338  244111 registry.go:341] failed to create container pid=244782 cg=/system.slice/docker-da91b1a7154749ae351b2f7a
7d70af8279528b1e0cd4dd8fbe9ca68060561f5b.scope id=/swarm/system-prune/system-prune/rq437f964bxm6zbf3d9zfst9v: no such file or directo
ry
W0519 14:43:32.621338  244111 registry.go:341] failed to create container pid=244782 cg=/system.slice/docker-da91b1a7154749ae351b2f7a
7d70af8279528b1e0cd4dd8fbe9ca68060561f5b.scope id=/swarm/system-prune/system-prune/rq437f964bxm6zbf3d9zfst9v: no such file or directo
ry
I0519 14:43:32.622423  244111 registry.go:314] calculated container id 244783 -> /system.slice/docker-da91b1a7154749ae351b2f7a7d70af8
279528b1e0cd4dd8fbe9ca68060561f5b.scope -> /swarm/system-prune/system-prune/rq437f964bxm6zbf3d9zfst9v
I0519 14:43:32.622694  244111 container.go:911] "started container logparser" cg="/system.slice/docker-da91b1a7154749ae351b2f7a7d70af
8279528b1e0cd4dd8fbe9ca68060561f5b.scope"
 
 
I0519 14:44:03.981621  244111 registry.go:314] calculated container id 245494 -> /system.slice/docker-67d75ce828cebdf472081eeeca71890
0f58b08e50b693bdcbccd620268036d84.scope -> /swarm/portainer/agent/rq437f964bxm6zbf3d9zfst9v
W0519 14:44:03.981674  244111 registry.go:341] failed to create container pid=245494 cg=/system.slice/docker-67d75ce828cebdf472081eee
ca718900f58b08e50b693bdcbccd620268036d84.scope id=/swarm/portainer/agent/rq437f964bxm6zbf3d9zfst9v: no such file or directory
W0519 14:44:03.981674  244111 registry.go:341] failed to create container pid=245494 cg=/system.slice/docker-67d75ce828cebdf472081eee
ca718900f58b08e50b693bdcbccd620268036d84.scope id=/swarm/portainer/agent/rq437f964bxm6zbf3d9zfst9v: no such file or directory
I0519 14:44:03.982908  244111 registry.go:314] calculated container id 245495 -> /system.slice/docker-67d75ce828cebdf472081eeeca71890
0f58b08e50b693bdcbccd620268036d84.scope -> /swarm/portainer/agent/rq437f964bxm6zbf3d9zfst9v
I0519 14:44:03.983397  244111 container.go:911] "started container logparser" cg="/system.slice/docker-67d75ce828cebdf472081eeeca7189
00f58b08e50b693bdcbccd620268036d84.scope"

**Comment by ovizii** [comment]
Did you ever figure it out?
